bpfman/Cargo.toml

```diff
@@ -6,9 +6,13 @@ name = "bpfman"
 repository.workspace = true
 version.workspace = true
 
+[lib]
+name = "bpfman"
+path = "src/lib/bpf.rs"
+
 [[bin]]
 name = "bpfman"
-path = "src/main.rs"
+path = "src/cli/main.rs"
```

Comment:
By convention, this is usually `src/bin/bpfman.rs` or `src/bin/main.rs`

Reply:
Hrm based on the convention here (https://doc.rust-lang.org/cargo/guide/project-layout.html) I'd like to make it /src/bin/cli/main.rs since this is a "multi-file-executable)

Reply:
Additionally `/src/bin/main.rs` `/src/bin/used_by_main.rs` just won't work